source/api_cc/tests/CMakeLists.txt

```diff
@@ -76,6 +76,45 @@ if (USE_CUDA_TOOLKIT)
   add_subdirectory(${LIB_BASE_DIR}/src/cuda cuda_binary_dir)
 endif()
 
+# define USE_ROCM_TOOLKIT
+if(NOT DEFINED HIP_PATH)
+  if(NOT DEFINED ENV{HIP_PATH})
+    set(HIP_PATH "/opt/rocm/hip" CACHE PATH "Path to which HIP has been installed")
```

Comment:
I do not understand why here we need hard coding.

Reply:
Because it's only when I give this absolute path that it can find the HIP package

Reply:
![image](https://user-images.githubusercontent.com/47965866/119253825-9cf2e100-bbe5-11eb-8fd4-5d6b0eb6ce6a.png)

Reply:
If I comment out this code, it makes error  when I execute "make"

Reply:
I mean the path to hip should be discovered by `find_package(HIP)`

Reply:
Get, maybe I should find a macro in the FindHIP.cmake to replace absolutely path

Reply:
We must do it. Otherwise we cannot support machines with hip installed to any other path.

Reply:
![image](https://user-images.githubusercontent.com/47965866/119599570-d6ab2e00-be17-11eb-9802-0f431d2367e9.png)

Reply:
I haven't found HIP-related macros or files on cmake 3.15，It is possible that cmake packages findHIP as a binary, thereby inheriting the older HIP